src/ngraph/op/replace_slice.cpp

```diff
@@ -176,11 +176,10 @@ void op::ReplaceSlice::generate_adjoints(autodiff::Adjoints& adjoints, const Nod
 {
     auto delta = deltas.at(0);
 
-    auto x = get_inputs().at(0).get_output().get_node();
-    auto& y_input = get_inputs().at(1);
-    auto y = y_input.get_output().get_node();
-    auto& y_element_type = y_input.get_element_type();
-    auto y_shape = y_input.get_shape();
+    auto x = get_argument(0);
```

Comment:
OMG I can actually read this. What sorcery is this? :)

Reply:
In fairness, it could also have been written more readably with the old APIs :)